examples/python/src/main/java/com/hazelcast/jet/examples/python/Python.java

```diff
@@ -0,0 +1,109 @@
+import com.hazelcast.jet.pipeline.test.TestSources;
+import com.hazelcast.jet.python.PythonServiceConfig;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Stream;
+
+import static com.hazelcast.jet.impl.util.Util.uncheckRun;
+import static com.hazelcast.jet.python.PythonTransforms.mapUsingPython;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * This example shows you how to invoke a Python function to process the
+ * data in the Jet pipeline. The function gets a batch of items in a list
+ * and must return a list of result items.
+ * <p>
+ * The provided code uses {@code src/main/resources/python} in this project
+ * as the Python project directory. There are two files there:
+ * {@code requirements.txt} that declares NumPy as a dependency and
+ * {@code take_sqrt.py} that defines the {@code transform_list} function
+ * that Jet will call with the pipeline data. The function uses uses the
+ * NumPy library to transform the input list by taking the square root of
+ * each element.
+ */
+public class Python {
+
+    private static final String RESULTS = "python_results";
+
+    private static Pipeline buildPipeline(String baseDir) {
+        Pipeline p = Pipeline.create();
+        p.readFrom(TestSources.itemStream(10, (ts, seq) -> bigRandomNumberAsString()))
+         .withoutTimestamps()
+         .apply(mapUsingPython(new PythonServiceConfig()
+                 .setBaseDir(baseDir)
+                 .setHandlerModule("take_sqrt")))
+         .setLocalParallelism(1) // controls how many Python processes will be used
+         .writeTo(Sinks.observable(RESULTS));
+        return p;
+    }
+
+    private static String bigRandomNumberAsString() {
+        ThreadLocalRandom rnd = ThreadLocalRandom.current();
+        return "" + rnd.nextLong(Long.MAX_VALUE) + rnd.nextLong(Long.MAX_VALUE)
+                + rnd.nextLong(Long.MAX_VALUE) + rnd.nextLong(Long.MAX_VALUE);
+    }
+
+    public static void main(String[] args) throws IOException {
+        Path baseDir = Files.createTempDirectory("jet-python-sample-");
+        copyClasspathResourcesToDirectory("python", baseDir);
+        Pipeline p = buildPipeline(baseDir.toString());
+
+        JetInstance jet = Jet.bootstrappedInstance();
+        try {
+            Observable<String> observable = jet.getObservable(RESULTS);
+            observable.addObserver(Observer.of(System.out::println));
+            JobConfig config = new JobConfig().setName("python-mapping");
+            jet.newJobIfAbsent(p, config).join();
+        } finally {
+            IOUtil.delete(baseDir);
+            Jet.shutdownAll();
+        }
+    }
+
+    private static void copyClasspathResourcesToDirectory(String sourcePath, Path destPath) throws IOException {
```

Comment:
This seem rather like boilerplate and a lot of code.. could we move it somewhere else perhaps? as a util method on Python?